### DISCUSSION TITLE
Details on available towns

### DISCUSSION THREAD

**Discussion by qhaas** [carla-simulator/carla#4573] [Q&A]
Greetings,

Are the [various available towns documented in detail](https://carla.readthedocs.io/en/0.9.12/core_map/#non-layered-maps)?  Would be interesting to know things like surface area of driveable streets, carrying capacity of pedestrians (i.e. max number of walkers), etc.

Thanks

**Comment by markcoletti** [comment]
I second this because I need to set the fog distance, and it would be helpful to know the maximum width of a given town.
